Example/App/Sources/Networking/Interceptors/LinkOverrideInterceptor.swift

```diff
@@ -0,0 +1,24 @@
+// Copyright © 2022 Jamit Labs GmbH. All rights reserved.
+
+import Foundation
+import Jetworking
+
+/// This is an example of an interceptor that can be used to override the request url.
+public final class LinkOverrideInterceptor: Interceptor {
+    // MARK: - Properties
+    static let `default` = LinkOverrideInterceptor()
+    var overrideUrl: String? = nil
```

Comment:
Could the property type be changed to `URL` (otherwise it should be renamed, as it is not an `URL` but a `String`)?

```suggestion
    var overrideUrl: URL?
```